candle-transformers/src/models/llava/constants.rs

```diff
@@ -0,0 +1,7 @@
+//pub const IGNORE_INDEX: i32 = -100;
```

Comment:
Same as above, better not to keep commented out code.

Reply:
Same as above